src/Core/src/Handlers/Image/ImageHandler.Windows.cs

```diff
@@ -40,24 +38,31 @@ protected override void DisconnectHandler(FrameworkElement nativeView)
 			_sourceManager.Reset();
 		}
 
+		public static void MapBackgroundColor(ImageHandler handler, IImage image)
+		{
+			handler.UpdateValue(nameof(IViewHandler.ContainerView));
+
+			handler.ContainerView?.UpdateBackgroundColor(image);
+		}
```

Comment:
But, as a result, all the properties that need to work on the container are now redirected to the container instead of the default version that used the `NativeView`. In addition, we also have to let the handler know that how we might be changing the container, so first fire that and then we will have the container view to use.

I somehow feel that this could be changed... I was thinking that if the view has a container, then the base `ViewHandler` would use that for all the properties - for example: enabled, a11y, width, height - but this now means that if I add a background to my view, all the properties will need to be set on the new container and unset on the view. And it may be the case that setting a a11y property on a button handles differently when set on a border that contains a button. So for now, overriding the specific properties that need the container is not too bad. Maybe we need a way to mark properties (per handler) as a "this is a container property" and then if any of those are set the handler automatically creates a container.

Reply:
For A11y, there is [accessibilityTraits](https://developer.apple.com/documentation/objectivec/nsobject/1615202-accessibilitytraits) on iOS and I don't think there is an equivalent concept for Android. For width and height, the container view should always be the same as the child view, no?